test_wp_connection.py

```
@@ -0,0 +1,47 @@
+load_dotenv()
+
+# Add current directory to path
+sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)))
+
+from publisher import Publisher
+
+def test_connection():
+    """Test WordPress connection with current credentials"""
+    try:
+        # Create publisher instance
+        wp = Publisher()
```

Ruff (0.12.2)

Local variable `wp` is assigned to but never used

Remove assignment to unused variable `wp`

(F841)